dev-vcs/fuel/fuel-1.0.1.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="A QT Fossil GUI"
+DESCRIPTION=" Fuel is cross-platform GUI front-end for the excellent Fossil SCM tool \
+written in Qt. Fuel runs on Windows, OSX as well as various flavors of Unix including \
+Linux and FreeBSD"
+HOMEPAGE="https://fuel-scm.org/fossil/home/"
+COPYRIGHT="2018 Kostas Karanikolas"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://fuel-scm.org/files/releases/fuel-$portVersion.tar.gz"
+CHECKSUM_SHA256="4e463cb9f3cf9c31c7ae95ce76b558502c0b07e1dd4109da714997f0dc9a65db"
+PATCHES="fuel-1.0.1.patchset"
+SOURCE_FILENAME="fuel-$portVersion.tar.gz"
+SOURCE_DIR="fuel-$portVersion"
+ADDITIONAL_FILES="fuel.rdef.in"
+
+ARCHITECTURES="x86 x86_64"
```

Comment:
```diff
 CHECKSUM_SHA256="4e463cb9f3cf9c31c7ae95ce76b558502c0b07e1dd4109da714997f0dc9a65db"
-PATCHES="fuel-1.0.1.patchset"
 SOURCE_FILENAME="fuel-$portVersion.tar.gz"
-SOURCE_DIR="fuel-$portVersion"
+PATCHES="fuel-$portVersion.patchset"
 ADDITIONAL_FILES="fuel.rdef.in"
 
-ARCHITECTURES="x86 x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
 if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
 fi
```
(`SOURCE_DIR` is not needed here because its default value is correct.)

Reply:
..sorry..  ok changed